cmd/podman/buildfarm/remove.go

```diff
@@ -0,0 +1,92 @@
+package buildfarm
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/podman/v4/cmd/podman/common"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildfarmRmDescription = `Remove one or more existing farms.`
+	rmCommand              = &cobra.Command{
+		Use:               "remove [options] [FARM...]",
+		Aliases:           []string{"rm"},
+		Short:             "Remove one or more farms",
+		Long:              buildfarmRmDescription,
+		RunE:              rm,
+		ValidArgsFunction: common.AutoCompleteFarms,
+		Example: `podman buildfarm rm myfarm1 myfarm2
+  podman buildfarm rm --all`,
+	}
+
+	// Temporary struct to hold cli values.
+	rmOpts = struct {
+		All bool
+	}{}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: rmCommand,
+		Parent:  buildfarmCmd,
+	})
+	flags := rmCommand.Flags()
+	flags.BoolVarP(&rmOpts.All, "all", "a", false, "Remove all farms")
+}
+
+func rm(cmd *cobra.Command, args []string) error {
+	cfg, err := config.ReadCustomConfig()
+	if err != nil {
+		return err
+	}
+
+	if rmOpts.All {
+		cfg.Farms.List = make(map[string][]string)
+		cfg.Farms.Default = ""
+		if err := cfg.Write(); err != nil {
+			return err
+		}
+		fmt.Println("All farms have been deleted")
```

Comment:
No need for output

Reply:
Should we just print the farm name similar to what we do for images and containers when they are removed.